flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/TestUtils.java

```diff
@@ -397,5 +398,10 @@ public ManagedDependentResourceContext managedDependentResourceContext() {
         public EventSourceRetriever<T> eventSourceRetriever() {
             return null;
         }
+
+        @Override
+        public KubernetesClient getClient() {
+            throw new UnsupportedOperationException("Not implemented");
+        }
```

Comment:
Why do we need this?

Reply:
The josdk Context interface added a new method to access the client, we need to implement it in our tests but we are not using it yet.